doc/avocado_predictors_report.Rmd

```diff
@@ -39,27 +41,26 @@ The features we tested were:
 - `region`: U.S. state avocado was sold in
 - `month`: month avocado was sold in
 
-The features we used in the random forest model were:
+The features we used in the random forest regression model were:
 
 - `type`: type of avocado sold (conventional or organic)
-- `year`: year avocado was sold in
 - `region`: U.S. state avocado was sold in
 - `month`: month avocado was sold in
 
 The target was:
   
 - `average_price`: average price of avocado sold
 
-To perform this anlaysis, the R and Python programming languages [@r; @Python] as well as the following R and Python packages were used: caret [@caret], docopt [@docopt], feather [@feather], tidyverse [@tidyverse], lubridate [@lubridate], corr [@corr], car [@car], kableExtra [@kableExtra], reshape2 [@reshape2], tidyverse [@tidyverse], broom [@broom], knitr [@knitr], ADD PACKAGES USED FOR ML.
+To perform this anlaysis, the R and Python programming languages [@r; @Python] as well as the following R and Python packages were used: caret [@caret], docopt [@docopt], feather [@feather], tidyverse [@tidyverse], lubridate [@lubridate], corr [@corr], car [@car], kableExtra [@kableExtra], reshape2 [@reshape2], tidyverse [@tidyverse], broom [@broom], knitr [@knitr], pandas [@pandas], numpy[@numpy], scikit-learn[@scikit-learn], and altair[@2018-altair].
```

Comment:
I also had to install the `pyarrow` and `selenium` Python packages as well as `chromedriver` at the OS level.

Reply:
We also need to add the `here` R package.

Reply:
ok I will add those ones as well

Reply:
I can't find the bibtex for chromedriver or pyarrow

Reply:
Why don't we just merge what we have for now, and then you guys can work on the final report as well and I can see if I can find those references

Reply:
i tried looking and i can't find it either...

Reply:
added here package

Reply:
ok, sounds good.